config-model/src/main/java/com/yahoo/vespa/model/content/storagecluster/FileStorProducer.java

```diff
@@ -56,10 +58,15 @@ private Integer getThreads(ModelElement clusterElem) {
         }
     }
     public FileStorProducer(ModelContext.Properties properties, ContentCluster parent, Integer numThreads) {
+        final int twoMB = 0x200000;
+        final int fourK = 0x1000;
         this.numThreads = numThreads;
         this.cluster = parent;
         this.reponseNumThreads = properties.defaultNumResponseThreads();
         this.responseSequencerType = convertResponseSequencerType(properties.responseSequencerType());
+        useAsyncMessageHandlingOnSchedule = properties.useAsyncMessageHandlingOnSchedule();
+        mergeChunkSize = ((properties.mergeChunkSize() + (twoMB-1))/twoMB)*twoMB - fourK;
```

Comment:
Perhaps add a comment here on why we do the extra calculation.

Reply:
Fixed